app/src/main/java/com/climbz/strongcode/ConditioningActivity.java

```diff
@@ -0,0 +1,54 @@
+package com.climbz.strongcode;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConditioningActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_conditioning);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        //my code below
+        final ListView listview = (ListView) findViewById(R.id.listview);
+
+        List<Exercise> exercises = null;
+        try {
+            XMLExerciseParserHandler parser = new XMLExerciseParserHandler();
```

Comment:
using Gson to create objects is much easier then manually parsing xml, hit me up and will explain